js/wp-autoupdates.js

```diff
@@ -2,114 +2,151 @@
+					type = $anchor.attr( 'data-wp-type' ),
+					action = $anchor.attr( 'data-wp-action' ),
+					$label = $anchor.find( '.label' ),
+					$parent = $anchor.parents(
+						'themes' !== pagenow
+							? '.autoupdates_column'
+							: '.theme-autoupdate'
+					);
 
-					// Show loading status.
-					$label.text( 'enable' === action ? wp_autoupdates.enabling : wp_autoupdates.disabling );
-					$anchor.find( '.dashicons-update' ).removeClass( 'hidden' );
+				event.preventDefault();
 
-					data = {
-						action: 'toggle_auto_updates',
-						_ajax_nonce: settings.ajax_nonce,
-						state: action,
-						type: type,
-						asset: $anchor.attr( 'data-wp-asset' ),
-					};
+				// Clear any previous errors.
+				$parent
+					.find( '.auto-updates-error' )
+					.removeClass( 'notice error' )
+					.addClass( 'hidden' );
 
-					$.post( ajaxurl, data )
-					.done(
-						function( response ) {
-							var $enabled, $disabled, enabledNumber, disabledNumber;
+				// Show loading status.
+				$label.text(
+					'enable' === action
+						? wp_autoupdates.enabling
+						: wp_autoupdates.disabling
+				);
+				$anchor.find( '.dashicons-update' ).removeClass( 'hidden' );
 
-							if ( response.success ) {
-								  // Update the counts in the enabled/disabled views if on on
-								  // screen with a list table.
-								  // TODO: If either count started out 0 the appropriate span won't
-								  //       be there and hence won't be updated.
-								if ( 'themes' !== pagenow ) {
-									$enabled       = $( '.autoupdate_enabled span' );
-									$disabled      = $( '.autoupdate_disabled span' );
-									enabledNumber  = parseInt( $enabled.text().replace( /[^\d]+/g, '' ) ) || 0;
-									disabledNumber = parseInt( $disabled.text().replace( /[^\d]+/g, '' ) ) || 0;
+				data = { // eslint-disable-line
+					action: 'toggle_auto_updates',
+					_ajax_nonce: settings.ajax_nonce,
+					state: action,
+					type,
+					asset: $anchor.attr( 'data-wp-asset' ),
+				};
 
-									switch ( action ) {
-										case 'enable':
-											++enabledNumber;
-											--disabledNumber;
+				$.post( ajaxurl, data ) // eslint-disable-line no-undef
+					.done( function( response ) {
+						let $enabled, $disabled, enabledNumber, disabledNumber;
 
-											  break;
-										case 'disable':
-											--enabledNumber;
-											++disabledNumber;
-										 break;
-									}
+						if ( response.success ) {
+							// Update the counts in the enabled/disabled views if on on
+							// screen with a list table.
+							// TODO: If either count started out 0 the appropriate span won't
+							//       be there and hence won't be updated.
+							if ( 'themes' !== pagenow ) {
+								$enabled = $( '.autoupdate_enabled span' );
+								$disabled = $( '.autoupdate_disabled span' );
+								enabledNumber =
+									parseInt(
+										$enabled.text().replace( /[^\d]+/g, '' )
+									) || 0;
+								disabledNumber =
```

Comment:
Is there any way to disable the line length check for this statement and the one above it?  It makes it **really** hard to see the similarities in them (i.e., they are the same except for the variable names).  And to make it even worse, the 1st statement gets formatted into 4 lines, and the 2nd into 6 lines :-(

Reply:
Depending on your perspective this is either a joy or a frustration of Prettier, it handles all of this automatically 😉

Reply:
Yeah, let's table this. For now, let's let prettier do its thing.